### ISSUE TITLE
[Datumaro Project Export Error] lxml.etree.XPathSyntaxError: Invalid expression

### ISSUE THREAD

**Issue by AhmadM-DL** [opencv/cvat#1298]
I exported a datumaro dataset and tried to export it using the following:

``` datum --loglevel=debug project export -e "/item/annotation" --filter-mode 'i+a' -f yolo ```

However I got the following output: 

```
2020-03-22 14:52:40,358 DEBUG: Popen(['git', 'init'], cwd=C:\Users\PC\Downloads\t, universal_newlines=False, shell=None, istream=None)
2020-03-22 14:52:40,436 DEBUG: Failed checking if running in CYGWIN due to: FileNotFoundError(2, 'The system cannot find the file specified', None, 2, None)
2020-03-22 14:52:40,444 DEBUG: Imported the following symbols from cvat_rest_api_task_images.py: cvat_rest_api_task_images
2020-03-22 14:52:40,447 INFO: Loading the project...
2020-03-22 14:52:40,582 DEBUG: Loading 'task_11_images' source contents...
2020-03-22 14:52:40,624 DEBUG: Loading own dataset...
2020-03-22 14:52:40,675 INFO: Exporting the project...
2020-03-22 14:52:40,675 ERROR: Invalid expression
Traceback (most recent call last):
  File "C:\Program Files\Python37\Scripts\datum-script.py", line 11, in <module>
    load_entry_point('datumaro==0.1.0', 'console_scripts', 'datum')()
  File "c:\program files\python37\lib\site-packages\datumaro\cli\__main__.py", line 156, in main
    return args.command(args)
  File "c:\program files\python37\lib\site-packages\datumaro\cli\contexts\project\__init__.py", line 354, in export_command
    **filter_args)
  File "c:\program files\python37\lib\site-packages\datumaro\components\project.py", line 683, in export_project
    remove_empty=remove_empty)
  File "c:\program files\python37\lib\site-packages\datumaro\components\project.py", line 392, in extract
    remove_empty)
  File "c:\program files\python37\lib\site-packages\datumaro\components\extractor.py", line 710, in transform
    return method(self, *args, **kwargs)
  File "c:\program files\python37\lib\site-packages\datumaro\components\dataset_filter.py", line 223, in __init__
    xpath = ET.XPath(xpath)
  File "src\lxml\xpath.pxi", line 421, in lxml.etree.XPath.__init__
lxml.etree.XPathSyntaxError: Invalid expression
```

If I remove the ``` -e '/item/annotation'``` part I got the following error:

```
2020-03-22 14:57:55,646 DEBUG: Popen(['git', 'init'], cwd=C:\Users\PC\Downloads\t, universal_newlines=False, shell=None, istream=None)
2020-03-22 14:57:55,721 DEBUG: Failed checking if running in CYGWIN due to: FileNotFoundError(2, 'The system cannot find the file specified', None, 2, None)
2020-03-22 14:57:55,732 DEBUG: Imported the following symbols from cvat_rest_api_task_images.py: cvat_rest_api_task_images
2020-03-22 14:57:55,735 INFO: Loading the project...
2020-03-22 14:57:55,890 DEBUG: Loading 'task_11_images' source contents...
2020-03-22 14:57:55,933 DEBUG: Loading own dataset...
2020-03-22 14:57:55,985 INFO: Exporting the project...
2020-03-22 14:57:55,988 ERROR: 'Image' object has no attribute 'shape'
Traceback (most recent call last):
  File "C:\Program Files\Python37\Scripts\datum-script.py", line 11, in <module>
    load_entry_point('datumaro==0.1.0', 'console_scripts', 'datum')()
  File "c:\program files\python37\lib\site-packages\datumaro\cli\__main__.py", line 156, in main
    return args.command(args)
  File "c:\program files\python37\lib\site-packages\datumaro\cli\contexts\project\__init__.py", line 354, in export_command
    **filter_args)
  File "c:\program files\python37\lib\site-packages\datumaro\components\project.py", line 689, in export_project
    converter(dataset, save_dir)
  File "c:\program files\python37\lib\site-packages\datumaro\plugins\yolo_format\converter.py", line 81, in __call__
    height, width = item.image.size
  File "c:\program files\python37\lib\site-packages\datumaro\util\image.py", line 208, in size
    self._size = data.shape[:2]
AttributeError: 'Image' object has no attribute 'shape'
```

**Comment by zhiltsov-max** [comment]
I couldn't reproduce the first problem. Please check if you can do just:
```python
from lxml import etree as ET
ET.Xpath("/item/annotation")
```

The second issue should probably be fixed by this PR: #1301

**Comment by AhmadM-DL** [comment]
Regarding the first issue:
The following worked normally in a python script:
```
from lxml import etree as ET
ET.Xpath("/item/annotation")
```
Regarding the second issue:
When I check out to this pull request #1301  I end up facing the following problem: #1264 
Where the nginx default page is displayed on localhost and localhost/admin is available.

So I couldn't check whether  #1301 solved the problem or not as I couldn't export a new datumaro project as the cvat-ui is not available. ( I am using windows 7 and docker toolbox)

My repo is originally cloned from the develop branch earlier. Switching to the master also raised the
#1264 issue. However, moving back to an older version (cloned a couple of weeks ago) of  the develop branch resolves #1264 problem.

I am starting to lose hope from getting this working on windows seven, I think I will export my annotations and move to ubuntu on a VM machine.

**Comment by zhiltsov-max** [comment]
Can't reproduce the problem. #1352 adds an error message to check Xpath separately. Closing, feel free to reopen.
